library/utility/ftrack/library/utility/url/checker.py

```diff
@@ -0,0 +1,69 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+from urllib.parse import urlparse, urlunparse
+
+
+# Generic URL Checker Dispatcher
+def url_checker(url: str, checkers: list) -> str:
```

Comment:
If we have a function like this, we should use it and implement our ftrack checkers to be provided to this function.

However, the functionality seems generic enough, that python's builtin functionality should be more than enough:
```
url_valid = all([check_function(url) for check_function in checker_functions])
```

Reply:
Interesting, lets talk about this.

Reply:
Lets remove the url_checker and use the ftrack_url_checker everywhere.